File: app/src/main/java/ch/hearc/kumoslife/shop/Food.java
package ch.hearc.kumoslife.shop;

public class Food extends Item
{
    private double nutritiveValue;
    public Food(String name, double prize, double nutritiveValue) {
        super(name, prize);
        this.nutritiveValue = nutritiveValue;
    }

    public double getNutritiveValue()
    {
        return nutritiveValue;
    }

    @Override
    public String getInfos() {
        return "Nutritive value : " + nutritiveValue;
    }
}

File: app/src/main/java/ch/hearc/kumoslife/shop/Food.kt
package ch.hearc.kumoslife.shop

class Food(name: String?, prize: Double, nutritiveValue: Double, imageId: Int, ) : Item(name, prize, imageId) {
    private val nutritiveValue: Double?
    init {
        this.nutritiveValue = nutritiveValue
    }
    override fun info(): String {
        return "Nutritive Value: $nutritiveValue"
    }
}